pathod/pathod.py

```diff
@@ -284,10 +285,10 @@ def addlog(self, log):
         # moment because JSON encoding can't handle binary data, and I don't
         # want to base64 everything.
         if self.server.logreq:
-            encoded_bytes = self.rfile.get_log().encode("string_escape")
+            encoded_bytes = strutils.bytes_to_escaped_str(self.rfile.get_log())
```

Comment:
The comment about JSON there is out of date, since I've just ripped out the web interface entirely, and that was the only portion that required the logs to be JSON serializable. We now have a much simpler thread lock over the log and retrieve it directly. So, strip the out of date comment, and no need to escape the log data.